Description: Show list of users by default
 Just because a config entry says that it's false by default does not mean it
 actually is.
Author: Simon Quigley <tsimonq2@ubuntu.com>
Origin: vendor
Last-Update: 2025-01-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/data/lightdm.conf
+++ b/data/lightdm.conf
@@ -100,9 +100,9 @@
 #xdmcp-port=177
 #xdmcp-key=
 #greeter-session=example-gtk-gnome
-#greeter-hide-users=false
+greeter-hide-users=false
 #greeter-allow-guest=true
-#greeter-show-manual-login=false
+greeter-show-manual-login=false
 #greeter-show-remote-login=true
 #user-session=default
 #allow-user-switching=true
